src/components/Bridge/Bridge.tsx

```diff
@@ -1,28 +1,48 @@
 import { memo, useCallback, useEffect, useMemo, useState } from 'react';
 import { useConnectWallet } from '@web3-onboard/react';
-import { R_TOKEN, SUPPORTED_BRIDGE_NETWORKS, SupportedBridgeNetworks } from '@raft-fi/sdk';
+import { R_TOKEN, SUPPORTED_BRIDGE_NETWORKS, BRIDGE_NETWORK_LANES, SupportedBridgeNetwork } from '@raft-fi/sdk';
 import { Decimal } from '@tempusfinance/decimal';
 import { ButtonWrapper, TokenLogo } from 'tempus-ui';
 import { INPUT_PREVIEW_DIGITS, USD_UI_PRECISION } from '../../constants';
 import { formatCurrency } from '../../utils';
+import { MAINNET_NETWORKS, NETWORK_LOGO_VARIANTS, NETWORK_NAMES, TESTNET_NETWORKS } from '../../networks';
 import { useBridgeTokens, useWallet } from '../../hooks';
 import { CurrencyInput, ExecuteButton, Icon, Typography, ValueLabel } from '../shared';
 import PoweredBy from './PoweredBy';
 import NetworkSelector from './NetworkSelector';
 
 import './Bridge.scss';
-import { NETWORK_LOGO_VARIANTS, NETWORK_NAMES } from '../../networks';
 
 const Bridge = () => {
+  let defaultFromNetwork: SupportedBridgeNetwork;
+  let defaultToNetwork: SupportedBridgeNetwork;
+  if (import.meta.env.VITE_BRIDGE_ENVIRONMENT === 'mainnet') {
```

Comment:
think we need to have a config for testnet and mainnet and put all these there. can worry later.